ow_materials/include/Material.h

```diff
@@ -0,0 +1,44 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef MATERIAL_H
+#define MATERIAL_H
+
+#include <limits>
+
+namespace ow_materials
+{
+
+using MaterialID = uint8_t;
+
+struct Color
+{
+  uint8_t r, g, b;
```

Comment:
Use float or double so colors can be blended at high precision. They should only be reduced to uint8 when we want to use them in a texture map.

There is also a chance that material colors differ enough that we would require higher dynamic range and use a floating point texture.

There's also a chance that colors of blended materials would be derived from other properties rather than just directly mixing original colors.

Reply:
This is why I am glad to have your eyes on this. I'll change them to double, thanks.

Reply:
Changed to double